vertical-pod-autoscaler/pkg/recommender/input/cluster_feeder_test.go

```diff
@@ -119,6 +119,14 @@ func TestLoadPods(t *testing.T) {
 			expectedConfigUnsupported: nil,
 			expectedConfigDeprecated:  nil,
 		},
+		{
+			name:                      "no targetRef",
+			selector:                  parseLabelSelector("app = test"),
+			fetchSelectorError:        nil,
+			expectedSelector:          labels.Nothing(),
+			expectedConfigUnsupported: nil,
```

Comment:
I think it would be better if this was actually configUnsupported. Also VPA withouth targetRef is rejected by the admission controller so it shouldn't be possible to create an object like this.

Reply:
yes. But this will hit this line, and it's return condition is  default value. https://github.com/kubernetes/autoscaler/blob/de322e2034df4413381586d50d68620f5c222ed1/vertical-pod-autoscaler/pkg/recommender/input/cluster_feeder.go#L442

I believe this case raise code coverage of cluster_feeder, and it really works.

Reply:
Agreed, though I meant maybe we should actually change the code :) It doesn't have to be part of this PR though.

Reply:
Yeah, maybe I can help with this afterwards~